modules/mono/editor/GodotTools/GodotTools/Build/BuildOutputView.cs

```diff
@@ -315,21 +315,21 @@ private void IssuesListContextOptionPressed(IssuesContextMenuOption id)
             switch (id)
             {
                 case IssuesContextMenuOption.Copy:
-                {
-                    // We don't allow multi-selection but just in case that changes later...
-                    string text = null;
-
-                    foreach (int issueIndex in _issuesList.GetSelectedItems())
                     {
+                        // We don't allow multi-selection but just in case that changes later...
+                        string text = null;
+
+                        foreach (int issueIndex in _issuesList.GetSelectedItems())
+                        {
+                            if (text != null)
+                                text += "\n";
+                            text += _issuesList.GetItemText(issueIndex);
+                        }
+
                         if (text != null)
-                            text += "\n";
-                        text += _issuesList.GetItemText(issueIndex);
+                            OS.Clipboard = text;
+                        break;
                     }
-
-                    if (text != null)
-                        OS.Clipboard = text;
-                    break;
-                }
```

Comment:
It looks like your IDE indented this switch case. We have an [`.editorconfig`](https://github.com/godotengine/godot/blob/d52e604b01f5f9845be6d0c70ef1520827f1004e/modules/mono/.editorconfig#L14) to prevent this, but it seems like your IDE may not be configured to use it.